bin/main.rb

```diff
@@ -1,35 +1,58 @@
-# Set up game
-puts "Welcome!"
-puts "Player 1, What is your name?"
-@player1 = gets.chomp
-puts "Now, Player 2, What is your name?"
-@player2 = gets.chomp
-puts "Let's play! #{@player1} #{@player2}"
-count = 0
+require_relative  '../lib/game.rb'
+puts
+puts "*" * 27
+puts "  WELCOME TO TIC-TAC-TOE!"
+puts "*" * 27
+puts
+print "Player 1, What is your name?: "
+player1 = gets.chomp.capitalize
+print "Now, Player 2, What is your name?: "
+player2 = gets.chomp.capitalize
```

Comment:
The reason why we kept the player name inputs on the main was because you made it clear that we can't use "puts/gets" inside game logic. By game logic, do you just mean the game class? Because we understood that as keeping those very specific to the main.rb, since that is in essence user interaction.

Reply:
Yes, I understand. What if we use this method inside the `main.rb` file 🤔
What I was pointing out is the repetition.

Reply:
It doesn't have to be necessarily this method in the example above though. The above is only an example of how it (DRY) be achieved.

Reply:
Well darn, I thought you had gave us something helpful but it turns out that it's only made things more confusing.

Reply:
Oops, sorry about that. :disappointed: 
Can you explain what issue you having? Paste a snippet here if you don't mind.
Note that the example above is straightforward and I also tested it before suggesting. Using the above example, you need to remove the previous equivalent. code.